docs/setup/arch/systems/README.md

```diff
@@ -0,0 +1,102 @@
+
+> [!IMPORTANT]
+> FirecREST doesn't execute commands as `root` user or using `sudo` commands. All commands are executed on behalf of the users using the users' credential.
+
+## JWT to SSH Delegation
+
+To answer to the question on "how the SSH credentials are obtained or created", FirecREST provides an integration with JSON Web Tokens used to authenticate agains the API.
+
+FirecREST uses the `username` or `preferred_username` claim from the JWT access token created by the Identity Provider (IdP) when the user or application [authenticated to use the API](../auth/README.md).
+
+> [!IMPORTANT]
+> This `username` value on the token must be a valid username on the HPC system, otherwise the SSH credential created on its behalf won't be allowed in the system.
+
+The JWT is decoded by FirecREST using the IdP public key (checking that the token is valid in time and is from a trusted source), the `username` is extracted and the SSH key is created for the user and then sent to the cluster in the SSH connection.
+
+### Obtaining SSH credentials on behalf of the user
+
+FirecREST provides an abstraction to allow different ways of getting SSH credentials for the user. Currently, there are the 2 options available:
+
+[](){#f7t-ssh-service}
+
+1. Using an "SSH Service"
+
+    If the HPC center has an API or web service that translates a JWT into an SSH credential, you can connect FirecREST to this service.
+
+    ![f7t_ssh_service](../../../assets/img/command_exec_sshservice.svg)
+
+    In this case, FirecREST uses the service providing the JWT used to authenticate with FirecREST, therefore the SSH Service must trust on the IdP of FirecREST.
+
+    Once the SSH credential is created, FirecREST uses the SSH credential to execute commands.
+
+    The benefit of this approach is that SSH credentials creation and validation is managed within the specific groups.
+
+2. Using static SSH credentials
+
+    Additionally, FirecREST provides a way to map user identities in the JWT with SSH credentials by using the [firecrest-config.yaml](../../conf/README.md).
+
+    ![f7t_ssh_service](../../../assets/img/command_exec_nosshsvc.svg)
+
+    SSH credentials or the users should be managed as secrets and should be updated by the administrator of the FirecREST deployment if they change.
+
+## SSH Configuration in target systems
+
+> [!IMPORTANT]
+> All these configurations are optional, but strongly suggested to increase security (remember that we are allowing machines to machine communication via SSH, we don't want an SSH DoS!) and provide high througput regime via API
+
+It is recommended that FirecREST command execution has a specific `Match` section on the [Open SSH Configuration](https://man.openbsd.org/OpenBSD-current/man5/ssh_config.5#Match) of the target system:
```

Comment:
Maybe:
It is recommended to set a specific `Match` section on the [Open SSHD configuration] (https://man.openbsd.org/OpenBSD-current/man5/sshd_config.5#Match) for requests from FirecREST:

The link has to point to 'sshd_config' instead of 'ssh_config', the rule is similar but options differ.